core/js/scenario.class.js

```diff
@@ -580,14 +580,14 @@ jeedom.scenario.autoCompleteAction = [
   'popup',
   'event',
   'changeTheme',
+  'remove_inat',
```

Comment:
It would be beneficial to add a comment explaining why 'remove_inat' was moved from 'autoCompleteActionScOnly' to 'autoCompleteAction'. This would improve code readability and maintainability. [medium]

Reply:
`remove_inat` is now also usable outside of scenario, as it is can be linked to a specific scenario.